storage/remote/queue_manager.go

```diff
@@ -485,14 +544,13 @@ func (s *shards) runShard(i int) {
 			queueLength.WithLabelValues(s.qm.queueName).Dec()
 			pendingSamples = append(pendingSamples, sample)
 
-			if len(pendingSamples) >= s.qm.cfg.MaxSamplesPerSend {
-				s.sendSamples(pendingSamples[:s.qm.cfg.MaxSamplesPerSend])
-				pendingSamples = pendingSamples[s.qm.cfg.MaxSamplesPerSend:]
+			if len(pendingSamples) >= 100 {
```

Comment:
Don't hard code this, use cfg.

Reply:
same question, should this still be configurable? can we just have it as a constant value when creating a new shard?